lib/smart_answer_flows/marriage-abroad/outcomes/countries/cyprus/ceremony_country/partner_british/_same_sex.erb

```diff
@@ -0,0 +1,76 @@
+You may be able to register a civil partnership at the High Commission in Cyprus.
+
+Contact them to make an appointment:
+
+$A
+British High Commission Nicosia
+Alexander Pallis Street
+PO Box 21978
+Nicosia
+1587
+Cyprus
+$A
+
+$C
+Telephone: +357 22 861100
+Fax: +357 22 861125
+
+Email: <UKinCyprus@fco.gov.uk>
+
+[British High Commission Nicosia - opening hours](https://www.gov.uk/government/world/organisations/british-high-commission-nicosia/office/british-high-commission)
+$C
+
+$A
+Consular section
+Consular section
+Alexander Pallis Street
+PO Box 21978
+Nicosia
+1587
+Cyprus
+$A
+
+$C
+Telephone:  +357 22 861100
+Fax:  +357 22 861200
+
+Email: <UKinCyprus@fco.gov.uk>
+
+[Consular section - opening hours](https://www.gov.uk/government/world/organisations/british-high-commission-nicosia/office/consular-section-british-high-commission-nicosia)
+$C
+
+##What documents you’ll need
+
+You’ll need proof that you’ve been resident in the area covered by the High Commission for at least 7 days, like an employer’s letter or a bank statement.
+
+You’ll both need your original passports. If either of you have been divorced, widowed or in a civil partnership before, you’ll also need:
+
+- a [decree absolute or final order](/copy-decree-absolute-final-order), or
+- the [death certificate](/order-copy-birth-death-marriage-certificate/)
+
+###What you need to do
+
+Once you’ve made your appointment, the High Commission will give you:
+
+- a notice of registration
+- a declaration that you and your partner will need to swear, stating that you’re legally entitled to enter into a civil partnership
+
+Once you’ve submitted these and paid the registration fee (see below), the High Commission will display your notice publicly for 14 days.
+
+As long as nobody has registered an objection after this time, the registration officer can then register your partnership any time until 3 months after the date you gave notice.
+
+You’ll need to provide two witnesses and pay a fee to register your civil partnership. You’ll need to pay an additional fee for your civil partnership certificate (see below).
+
+## Fees
+
+Service | Fee
```

Comment:
We should still be using the fees partial: `lib/smart_answer_flows/marriage-abroad/outcomes/_consular_fees_table_items.govspeak.erb` iirc.